### PR TITLE
Update cr, sx, so & bf.m3u

### PR DESCRIPTION
**Pull request by Carlinhos027**

### CODE REVIEW

**Review by freearhey — DISMISSED** [review]

File: streams/bf.m3u [hunk 1]
```diff
@@ -1,7 +1,11 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="BF1.bf",BF1 (540p) [Not 24/7]
+https://547f72e6652371c3.mediapackage.us-east-1.amazonaws.com/out/v1/7e66f389bec3463dac81e8d55e680431/index_3.m3u8
```
Comment (Dum4G) [inline streams/bf.m3u:3]:
```suggestion
https://547f72e6652371c3.mediapackage.us-east-1.amazonaws.com/out/v1/7e66f389bec3463dac81e8d55e680431/index.m3u8
```

File: streams/cr.m3u [hunk 2]
```diff
@@ -13,26 +13,42 @@ https://59ef525c24caa.streamlock.net/canal1cr/canal1cr/playlist.m3u8
...
 #EXTINF:-1 tvg-id="Canal8.cr",Canal 8 (720p) [Not 24/7]
 https://mdstrm.com/live-stream-playlist/5a7b1e63a8da282c34d65445.m3u8
+#EXTINF:-1 tvg-id="CanalNueve.cr",Canal 9 (720p) [Geo-blocked] 
+http://201.201.149.86:8080/livestream/stream.m3u8
 #EXTINF:-1 tvg-id="Canal11.cr",Canal 11 (720p) [Not 24/7]
 https://d2t46xrpai3it5.cloudfront.net/Repretel11/b3e2798525101c45f84a48573676853a.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="TVN14.cr",Canal 14 San Carlos (720p) [Not 24/7]
 http://tvn.obix.tv:1935/TVN/CH14.stream_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal17TVNosara.cr",Canal 17 TV Nosara (720p)
+https://stmv.streamingvip.click/tvnosara2/tvnosara2/playlist.m3u8
```
Comment (Dum4G) [inline streams/cr.m3u:31]: Geo-blocked? Not 24/7 Couldn't verify

File: streams/cr.m3u [hunk 3]
```diff
@@ -65,6 +87,10 @@ http://tiquiciatv.com:1935/stv/web/playlist.m3u8
 https://lstv.duckdns.org:449/hls/stv.m3u8
 #EXTINF:-1 tvg-id="SOYPlanchaTV.cr",SOY Plancha TV (720p)
 https://59ef525c24caa.streamlock.net/vmtv/soyplancha/playlist.m3u8
+#EXTINF:-1 tvg-id="TDtv.cr",TDtv (720p)
+https://cloudvideo.servers10.com:8081/8230/index.m3u8
```
Comment (Dum4G) [inline streams/cr.m3u:91]: Got only sound, may be connection issue

File: streams/cr.m3u [hunk 4]
```diff
@@ -65,6 +87,10 @@ http://tiquiciatv.com:1935/stv/web/playlist.m3u8
...
+#EXTINF:-1 tvg-id="TeleBrunca.cr",TeleBrunca (720p)
+https://59ef525c24caa.streamlock.net/telebrunca/telebrunca/playlist.m3u8
```
Comment (Dum4G) [inline streams/cr.m3u:93]: 404

File: streams/cr.m3u [hunk 5]
```diff
@@ -102,3 +130,7 @@ https://59ef525c24caa.streamlock.net/vmtv/tvvintage/playlist.m3u8
 https://59ef525c24caa.streamlock.net/vmtv/vmlatino/playlist.m3u8
 #EXTINF:-1 tvg-id="VoiceOverRadioTV.cr",VoiceOver Radio TV (720p)
 https://cloudvideo.servers10.com:8081/8198/index.m3u8
+#EXTINF:-1 tvg-id="XpressoJovenRadio.cr",Xpresso Joven Radio (720p)
+https://stmv.streamingvip.click/xpressojovenradiotv/xpressojovenradiotv/playlist.m3u8
```
Comment (Dum4G) [inline streams/cr.m3u:134]: Geo-blocked? Not 24/7 Couldn't verify

Comment (Carlinhos027) [reply streams/cr.m3u:134]: Working for me

File: streams/cr.m3u (hunk 3 above)
Comment (Carlinhos027) [reply streams/cr.m3u:91]: Working for me

File: streams/cr.m3u (hunk 4 above)
Comment (Carlinhos027) [reply streams/cr.m3u:93]: Also doesn't work on its official site, so i think is Not 24/7

File: streams/cr.m3u (hunk 5 above)
Comment (Dum4G) [reply streams/cr.m3u:134]: Is Geo-blocked then

Comment (Carlinhos027) [reply streams/cr.m3u:134]: Not sure about that because I'm in Spain.

Comment (Dum4G) [reply streams/cr.m3u:134]: Nvm, it's antiRussian measure. Works in Japan and Germany

File: streams/cr.m3u [hunk 6]
```diff
@@ -65,6 +87,10 @@ http://tiquiciatv.com:1935/stv/web/playlist.m3u8
 https://lstv.duckdns.org:449/hls/stv.m3u8
 #EXTINF:-1 tvg-id="SOYPlanchaTV.cr",SOY Plancha TV (720p)
 https://59ef525c24caa.streamlock.net/vmtv/soyplancha/playlist.m3u8
+#EXTINF:-1 tvg-id="TDtv.cr",TDtv (720p)
+https://cloudvideo.servers10.com:8081/8230/index.m3u8
+#EXTINF:-1 tvg-id="TeleBrunca.cr",TeleBrunca (720p)
```
Comment (freearhey) [inline streams/cr.m3u:92]:
```suggestion
#EXTINF:-1 tvg-id="TeleBrunca.cr",TeleBrunca (720p) [Not 24/7]
```

image

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Comment by BellezaEmporium** [comment]
@Dum4G glad to see you back on track :)

**Review by freearhey — APPROVED** [review]

**Pull request merged**